xhprof/README.md

```diff
@@ -0,0 +1,17 @@
+Using XHProf with the WordPress core development environment
+============================================================
+
+[XHProf](https://www.php.net/manual/en/book.xhprof.php) is an open source PHP profile.
+
+### Setup with WordPress core development environment
+
+1. git clone https://github.com/WordPress/wordpress-develop.git
+2. Setup the environment using [these](https://github.com/wordPress/wordpress-develop/#development-environment-commands) instructions.
+3. Start environment, using `npm run env:start`
+4. Go to http://localhost:8889 and setup site.
+5. Stop environment, using `npm run env:stop`
+6. Copy docker-compose.override.yml into the root of wordpress-develop directory.
+7. Copy advanced-cache.php into the src/wp-content directory.
+8. Add `define( 'WP_CACHE', true );` to wp-config.php ( to enable profiler ).
```

Comment:
Is there a way we can somehow automate this via the `docker-compose.override.yml` file? Maybe with a custom command inside it?

Reply:
I didn't want to do this. I want developers to be able to bootstarp the profiler in how they want. I think that is a good idea for a v2 of this.

Reply:
@spacedmonkey Makes sense regarding the `WP_CACHE` constant in `wp-config.php`. But what would be the drawback of copying `advanced-cache.php` automatically for them?

Reply:
Where would it copy it from? Have it as part of the PHP docker container? 

Maybe like this - https://github.com/WordPress/wordpress-develop/blob/6222a118c9b87ca9f5703b5055831457469627b7/docker-compose.yml#L52-L53

If we do that, maybe we should just bootstrap the file in the wp-config.php file. If that profiler is bundled with docker image, then it makes it hard to control the config.

Reply:
@spacedmonkey Yeah, something along the lines of what you linked is what I was thinking.

I think either would work. The benefit of using `wp-config.php` would be that we don't have to "highjack" the `advanced-cache.php` file for something that is not an advanced cache. Let me know what you prefer.

Reply:
The profiler SDK is now part of the container. Start the profiler, is really up to you. I have provided this advanced-cache.php. Should we recommend bootstrapping in wp-config.php? 

To be clean, I would to have a profiler file accessilbe to the developer, so that can tweak it and config it.

Reply:
I have removed the advanced-cache.php workaround and just documented use in wp-config.php include.